packages/rest-generator/src/wrapper-files/api-file.ts

```diff
@@ -0,0 +1,95 @@
+import { codeBlock, unique } from '@sap-cloud-sdk/util';
+import { OpenApiDocument, OpenApiOperation } from '../openapi-types';
+
+/**
+ * @experimental This API is experimental and might change in newer versions. Use with caution.
+ * Get the content of the SAP Cloud SDK API wrapper.
+ * @param openApiDocument Parsed service.
+ * @returns The generated code for the SDK API wrapper.
+ */
+export function apiFile(openApiDocument: OpenApiDocument): string {
+  const requestBodyTypes = getRequestBodyTypes(openApiDocument);
+  return codeBlock`
+import { RestRequestBuilder } from '@sap-cloud-sdk/core';
+import { DefaultApi } from './openapi/api';
+${
+  requestBodyTypes
+    ? `import { ${requestBodyTypes} } from './openapi/model';`
```

Comment:
I guess you could use the `import type` here?  This never hurts if you just pass the type and could help avoid cyclic dependencies.

Reply:
I am also a bit afraid about stability here. Can we really rely on that there will be a file with the name in the models if the schema was named this way. What happens in case of naming conflict between a model schema type and a operation name. We will use a unique name and deviate from the initial name. If this is not an issue also explain why or if it is a valid concern create a follow up ticket.

Reply:
I think this is not an issue, but there are issues related to this: 
1. Why it is not an issue - we replace all tags in the original document before parsing. I found this weird but after some investigation I found that this is actually the only way to have one api for everything in one spec file. That tag is now 'default' so the name of the api is always `DefaultApi`, which is never exported on "package level"
2. Related issues - The stability around this whole wrapper is in fact questionnable. I have some todos left to check what the behavior of the openapi generator is in some cases. I already know of some incompatibilities that we already have. I will write them all down in a follow up - my concern is - have I considered all of them? ¯\\_(ツ)_/¯

Reply:
(BTW: you would have never seen this with ts-morph ;) )

Reply:
I cannot use import type because DefaultApi is also passed as the first parameter to the function.